Resources/Prototypes/Entities/Objects/Specific/Medical/hypospray.yml

```diff
@@ -49,6 +49,28 @@
   - type: UseDelay
     delay: 0.5
 
+- type: entity
+  name: experimental hypospray
+  suffix: Admeme
+  parent: BaseItem
+  description: The ultimate application of bluespace technology and rapid chemical administration.
+  id: AdminHypo
```

Comment:
All of this is straight up copypasted from the hypospray. Either make a BaseHypospray or at the very least parent from the regular one.

Reply:
Done